doc/classes/AudioServer.xml

```diff
@@ -198,6 +206,13 @@
 				Moves the bus from index [param index] to index [param to_index].
 			</description>
 		</method>
+		<method name="register_stream_as_sample" experimental="">
+			<return type="void" />
+			<param index="0" name="stream" type="AudioStream" />
+			<description>
+				Forces the registration of a stream as a sample. Lag spikes may occur when calling this method, especially on single-threaded builds. It is suggested to call this method while loading assets, where the lag spike could be masked, instead of relying on registering the sample when it needs to be played.
```

Comment:
These kind of long-winded sentences snuggly fit within a note because they:
- Do not describe the direct effect of the method
- Provide potential solutions or workarounds
```suggestion
				Forces the registration of a stream as a sample.
				[b]Note:[/b] Lag spikes may occur when calling this method, especially on single-threaded builds. It is suggested to call this method while loading assets, where the lag spike could be masked, instead of registering the sample right before it needs to be played.
```